src/vs/base/browser/ui/menu/menu.css

```diff
@@ -87,6 +87,10 @@
 	color: inherit;
 }
 
+.monaco-menu .monaco-action-bar.vertical .action-label.checked:after {
```

Comment:
@sbatten is this used currently anywhere?

Reply:
@bpasero this adds the checkmark to items that have been checked in the menu, like auto save